docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,259 @@ Postprocessing
+    :math:`0.04728`
+  - if the score is above :math:`0.5` predict 1
+
+The thresholds are actually the same for both groups here, which
+does not always have to be true. As a consequence, the only difference is
+the probability to predict label 1 in the case where the score is below
+:math:`0.5`.
+
+Note that the chart is different from the chart for :code:`equalized_odds`.
+It shows :code:`selection_rate` (which is the basis for our fairness
+criterion) on the x-axis :code:`demographic_parity`, and
+:code:`accuracy_score` (which is the basis for the implicit performance
+objective) on the y-axis.
+
+The following combinations of fairness criteria and objectives are available:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   *  - fairness criteria
+      - objectives
+   *  - :code:`equalized_odds`
+      - :code:`accuracy_score`, :code:`balanced_accuracy_score`
+   *  - :code:`demographic_parity`, :code:`false_positive_rate_parity`, :code:`false_negative_rate_parity`, :code:`true_positive_rate_parity`, :code:`true_negative_rate_parity`
+      - :code:`selection_rate`, :code:`true_positive_rate`, :code:`true_negative_rate`, :code:`accuracy_score`, :code:`balanced_accuracy_score`
+
+Using a pre-trained estimator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, :class:`ThresholdOptimizer` trains the passed estimator using its
+:code:`fit()` method. In some cases we already have a trained estimator, for
+example if we want to compare it with others. For this use case
+:class:`ThresholdOptimizer` has the :code:`prefit` argument. If set to
+:code:`True` :class:`ThresholdOptimizer` does not call :code:`fit()` on the
+estimator and assumes it is already trained.
+
+.. doctest:: mitigation
+
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from fairlearn.postprocessing import ThresholdOptimizer, plot_threshold_optimizer
+    >>> X = [[0], [1], [2], [3]]
+    >>> y = [0, 1, 0, 1]
+    >>> A = ['A', 'A', 'B', 'B']
+    >>> prefit_logistic_regression = LogisticRegression()
+    >>> prefit_logistic_regression.fit(X, y)
+    LogisticRegression()
+    >>> threshold_optimizer = ThresholdOptimizer(
+    ...     estimator=prefit_logistic_regression,
+    ...     constraints="demographic_parity",
+    ...     objective="accuracy_score",
+    ...     prefit=True)
+    >>> threshold_optimizer.fit(X, y, sensitive_features=A)
+    ThresholdOptimizer(estimator=LogisticRegression(), prefit=True)
+
+If it detects that the estimator
+may not be fitted as defined by
+`scikit-learn's check <https://scikit-learn.org/stable/modules/generated/sklearn.utils.validation.check_is_fitted.html>`_
+it prints a warning, but still proceeds with the assumption that the user
+knows what they are doing. Not every machine learning package adheres to
+scikit-learn's convention of setting all members with trailing underscore
+during :code:`fit()`, so this is unfortunately an imperfect check.
```

Comment:
This would change once we are compatible enough to put this estimator in a `GridSearchCV`, since that would mean calling `clone` on the estimator and the child estimator would not be fit then.

Reply:
True. Do you want me to mention that in some way? I feel like we can inform users that this option won't work inside a `GridSearchCV` when we are actually compatible, but I'm happy to add whatever you think makes sense.